fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeActivity.java

```diff
@@ -122,6 +122,10 @@ public void onBackPressed() {
     if (!(fragment instanceof SeedWalletFragment)) {
       setCurrentFragment(getSupportFragmentManager(), R.id.fragment_home, HomeFragment::new);
     }
+    // Exits the application if back is pressed on home
+    if (fragment instanceof HomeFragment) {
+      moveTaskToBack(true);
```

Comment:
The comment says that it exists the application, but the function says that it moves it. This sounds inconsistent. Isn't it a false exit, like the application is still opened but in the background ?

Reply:
Yeah it simply moves it in background, as it happens generally to every other mobile application, then you can close it through the task manager. With "exit" I was meaning just that, you exit the application but not close it

Reply:
Well, either properly close the application as in [](https://stackoverflow.com/questions/17719634/how-to-exit-an-android-app-programmatically) for example, and/or update the misleading comment. Exiting an application to me means that the application is closed and an exit code code is returned, but maybe I am too old fashion :-D